BridgeSDK/BridgeAPI/SBBUserManagerInternal.h

```diff
@@ -41,3 +42,9 @@ extern NSString * const kSBBUserDataGroupsAPI;
 
 extern NSString * const kSBBUserDataSharingScopeKey;
 extern NSString * const kSBBUserDataSharingScopeStrings[];
+
+@protocol SBBUserManagerInternalProtocol <SBBUserManagerProtocol>
+
+@property (nonatomic, strong) id<SBBActivityManagerInternalProtocol> activityManager;
```

Comment:
The user manager calls the activity manager to flush schedules when you update data groups. This is here in case needed for dependency injection for unit testing.